config/config.py

```
@@ -118,6 +122,11 @@ def validate_model(cls, v):
             warnings.warn(f"Using experimental model pattern: {v}. Please verify compatibility.", UserWarning)
             return v
 
+        # Ollama Cloud モデル: ollama/<name>[:<tag>] 形式（例: ollama/gemma4:31b-cloud）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)